frontend/src/metabase/query_builder/reducers.js

```diff
@@ -112,7 +112,6 @@ export const uiControls = handleActions(
         return {
           ...state,
           ...DEFAULT_UI_CONTROLS,
-          ...CLOSED_NATIVE_EDITOR_SIDEBARS,
```

Comment:
AlexP confirmed this line was redundant because it already reflects the state defined in `DEFAULT_UI_CONTROLS`.

[Slack thread](https://metaboat.slack.com/archives/C0645JP1W81/p1741206076121839).